tests/async/waitUntil.test.js

```diff
@@ -16,13 +16,21 @@ describe('waitUntil', () => {
     });
   });
 
-  it.skip('shuts down predicate after timeout', () => {
+  asyncIt.skip('shuts down predicate after timeout', () => {
     waitUntil(() => {
       console.log('this should stop logging after test times out');
     });
   });
 
-  it('handles exceptions, silently treating them as falsy return values', () => {
+  asyncIt.skip('handles expections, returning the message of the last one upon timeout', () => {
```

Comment:
Remove skipped test?

Reply:
Well no, this test checks that it fails with a nice user experience. But if I run them, they fail. So I don't know really, decided to keep them for documentation.